### ISSUE TITLE
Ability to configure keys for laptops without numpad

### ISSUE THREAD

**Issue by Tosyk** [FransBouma/ShaderToggler#9]
I don't have a numpad on my laptop keyboard and onscreen keyboard doesn't work.

**Comment by v00d00m4n** [comment]
yeah i was going to say the same, but i have pc with wireless keyboard+touchpad (instead of nampad) setup for console-like expirience with big tv, and hardcoded numpad buttons is always no way to go. 3d migoto that also cant dump and reinject shaders has similar process, but all buttons rebindable via ini, you can even bind xinput gamepad,  2nd gamepad if you have 2, which is way more comfortable. Maybe lookt at 3dmigoto code could help to make this task easy.

**Comment by liuxd17thu** [comment]
Try to migrate Alex's changes in REST back to ShaderToggler. Not sure if this one is well optimezed:
https://github.com/liuxd17thu/ReshadeEffectShaderToggler/tree/configurable_shortcut

**Comment by swapoer** [comment]
I found out that "ToggleKey=3707764736"  is "]".
